### PR TITLE
Updated Footer and Header

### PR DESCRIPTION
**Pull request by Alam9835**

### CODE REVIEW

File: api-docs/index.html [hunk 1]
```diff
@@ -149,85 +153,76 @@
...
-                                    <li><a href="https://www.joomla.org/core-features.html">Core Features</a></li>
-                                    <li><a href="https://www.joomla.org/about-joomla/the-project.html">The Project</a></li>
-                                    <li><a href="https://www.joomla.org/about-joomla/the-project/leadership-team.html">Leadership</a></li>
-                                    <li><a href="https://www.opensourcematters.org">Open Source Matters</a></li>
+
                                 </ul>
                             </li>
                             <li class="dropdown">
-                                <a class="dropdown-toggle" data-toggle="dropdown" href="#">Download &amp; Extend <span class="caret"></span></a>
+                                <a class="dropdown-toggle" data-toggle="dropdown" href="#">Download Extend  <span class="caret"></span></a>
```
Comment (HLeithner) [inline api-docs/index.html:180]: seems the & is missing now?

File: api-docs/index.html [hunk 2]
```diff
@@ -149,85 +153,76 @@
                         <ul id="nav-joomla" class="nav">
                             <li class="dropdown">
                                 <a class="dropdown-toggle" data-toggle="dropdown" href="#">
-                                    <span dir="ltr"><span aria-hidden="true" class="icon-joomla"></span> Joomla!<sup>&reg;</sup></span> <span class="caret"></span>
+                                    <span dir="ltr"><span aria-hidden="true" class="icon-joomla"></span> Joomla!<sup>®</sup></span> <span class="caret"></span>
```
Comment (HLeithner) [inline api-docs/index.html:156]: it seems you copied the browser version and not the source code, at least that's how it looks compared to joomla.org

File: api-docs/index.html (hunk 1 above)
Comment (Alam9835) [reply api-docs/index.html:180]: ok i will do it

File: api-docs/index.html (hunk 2 above)
Comment (Alam9835) [reply api-docs/index.html:156]: ok i will make the change

Comment (Alam9835) [reply api-docs/index.html:156]: else everything is fine or any more changes are needed?

**Comment by Alam9835** [comment]
I have updated please check.

**Comment by conconnl** [comment]
After reading a few lines, I think it's easier that I provide the complete file.
I will create a PR on your Repository to make it easier for everybody.

**Comment by conconnl** [comment]
Done, I updated your repository!

**Comment by Alam9835** [comment]
> Done, I updated your repository!

so can I make PR now?

**Comment by conconnl** [comment]
You already created the PR.
I was just waiting for you to check the difference with yours and the changes I applied to your branch. 

Any questions about it?
Else I will merge it.

**Comment by Alam9835** [comment]
ok please merge my PR

**Pull request merged**